utopia-remix/app/util/discord-webhook-node.d.ts

```diff
@@ -0,0 +1,69 @@
+declare module 'discord-webhook-node' {
```

Comment:
this is required since `discord-webhook-node` does not expose its types in an ESM compliant way

Reply:
That's disappointing

Reply:
yes, my initial thought was forking it and fixing its package.json (it's quite simple), but it seemed like an overkill for it